src/modules/user-settings/lib/editor.ts

```diff
@@ -6,6 +6,7 @@ export type SupportedEditorConfig = {
 	label: string;
 	url: ( path: string ) => string;
 	bundleId: string;
```

Comment:
```suggestion
	macOSBundleId: string;
```
Just a suggestion, but we might as well clarify that this is also platform-specific.

Reply:
Good point, done in b6528ade